heartbeat/monitors/active/http/http_test.go

```diff
@@ -55,7 +55,7 @@ func testRequest(t *testing.T, testURL string) *beat.Event {
 // an empty string no cert will be set.
 func testTLSRequest(t *testing.T, testURL string, extraConfig map[string]interface{}) *beat.Event {
 	configSrc := map[string]interface{}{
-		"urls":    testURL,
+		"hosts":   testURL,
```

Comment:
We still need to test the behavior of the `url` parameter, so we should add new test permutations here handling that.